client/odie/message/index.tsx

```diff
@@ -73,15 +107,57 @@ const ChatMessage = ( { message, isLast, messageEndRef }: ChatMessageProps ) =>
 				</div>
 			) }
 			{ message.type === 'introduction' && (
-				<>
-					<div className="odie-introduction-message-content">
-						<img src={ WapuuAvatar } alt="Wapuu profile" className="odie-chatbox-message-avatar" />
-						<div className="odie-chatbox-introduction-message">{ message.content }</div>
-					</div>
-				</>
+				<div className="odie-introduction-message-content">
+					<img src={ WapuuAvatar } alt="Wapuu profile" className="odie-chatbox-message-avatar" />
```

Comment:
Maybe translate the alt?

Reply:
Deferring this to a later PR as this might change too